onnxruntime/python/tools/transformers/fusion_rotary_attention.py

```diff
@@ -489,6 +602,26 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
             k_nodes = k_nodes_3
             past_k = concat_k.input[0]
             present_k = concat_k.output[0]
+        elif k_nodes_4 is not None and len(k_nodes_4) == 9:
+            logger.debug('fuse_rotary_attention: k_nodes_4')
+            logger.debug('*' * 30)
```

Comment:
Same comment to use `logger.debug` only when pattern matching fails

Reply:
done